regression/cbmc/Function_Pointer_Init/main.c

```diff
@@ -0,0 +1,10 @@
+#include <assert.h>
+
+typedef int (*other_function_type)(int n);
+
+void foo(other_function_type other_function)
+{
+  // returning from the function call is unreachable -> the following assertion
```

Comment:
:question: This comment doesn't make sense wrt to the test checking for verification failed?

Reply:
Extended.